Games/types/Mafia/roles/cards/Polariser.js

```diff
@@ -6,18 +6,31 @@ module.exports = class Polariser extends Card {
     super(role);
 
     this.meetings = {
-      Polarise: {
+      "Polarize Someone": {
         states: ["Night"],
-        flags: ["voting", "multi"],
+        flags: ["voting", "group", "speech", "multiActor"],
```

Comment:
multiactor means all polar bears visit
group means all polar bears vote together

you may refer to the documentation on what the flags mean